openflow_input/bsn_tlv

```diff
@@ -963,3 +963,16 @@ struct of_bsn_tlv_data_mask : of_bsn_tlv {
     uint16_t length;
     of_octets_t value;
 };
+
+enum ofp_bsn_port_usage(wire_type=uint16_t) {
```

Comment:
It's advisory to switchlight when allocating tx/rx buffers to physical interfaces. Larger buffers are more resilient against packet loss due to micorbursts.